VultisigApp/VultisigApp/View Models/SwapCryptoViewModel.swift

```diff
@@ -247,7 +247,7 @@ class SwapCryptoViewModel: ObservableObject, TransferViewModel {
                     coin: tx.fromCoin,
                     toAddress: toAddress ?? tx.fromCoin.address,
                     amount: tx.amountInCoinDecimal,
-                    memo: tx.quote?.memo,
+                    memo: getMemo(tx.quote?.memo, referralCode: referralCode),
```

Comment:
This is wrong , we should pass the affiliate address , and affiliate basis points to the quote endpoint ,  thus thorchain quote endpoint can construct correct memo for us for example

```
https://thornode.ninerealms.com/thorchain/quote/swap?from_asset=thor.rune&to_asset=thor.tcy&amount=100000000&destination=thor18altpx2gwt4c4ejr5uzda4kyzsudyn9q56fnng&streaming_interval=1&affiliate=amol&affiliate_bps=15&affiliate=vi&affiliate_bps=35
```
I believe we need to sort the affiliate address alphabetically , and and pass the affiliate_bps accordingly

Reply:
alternatively , you can revert these changes and address in a seperate PR